src/main/java/com/dataartisans/flink/dataflow/FlinkPipelineOptions.java

```diff
@@ -85,7 +85,8 @@
 	void setNumberOfExecutionRetries(Integer retries);
 
 	@Description("Sets the delay between executions. A value of {@code -1} indicates that the default value should be used.")
-	@Default.Long(-1L)
+	// TODO: Should really be -1 and the default set correctly.
+	@Default.Long(10L)
```

Comment:
I think we can change this back to -1 with the latest changes which set a default.